cedar-policy/src/api.rs

```diff
@@ -371,6 +359,19 @@ pub struct Diagnostics {
     errors: HashSet<String>,
 }
 
+impl From<authorizer::Diagnostics> for Diagnostics {
+    fn from(diagnostics: authorizer::Diagnostics) -> Self {
+        Self {
+            reason: diagnostics.reason.into_iter().map(PolicyId).collect(),
+            errors: diagnostics
+                .errors
+                .into_iter()
+                .map(|e| e.to_string())
```

Comment:
could be written `.map(ToString::to_string)`; not sure if that's clearer

Reply:
Fixed